UnityProject/Assets/Scripts/UI/Core/ChangeLogs/ChangeLogHandler.cs

```diff
@@ -11,23 +19,61 @@ public class ChangeLogHandler : MonoBehaviour
 		public Transform content;
 		private List<ChangeLogEntry> allEntries = new List<ChangeLogEntry>();
 		public GameObject entryPrefab;
+		public string ChangeLogLink = "https://changelog.unitystation.org/all-changes?format=json&limit=50";
+		public string ChangeLogParams = "all-changes?format=json&limit=50";
 
 		void Start()
 		{
-			window.SetActive(false);
+			if (string.IsNullOrEmpty(ChangeLogLink))
+			{
+				window.SetActive(false);
+				return;
+			}
+			GrabJsonData();
+		}
+
+		[Button("Get Data")]
+		private void GrabJsonData()
+		{
+			var client = new HttpClient();
+			client.BaseAddress = new Uri(ChangeLogLink);
```

Comment:
Can you leave the `BaseAddress` `null` and just pass the entire URL into `GetAsync`?